example/fake-provider-cf.yml

```diff
@@ -199,7 +199,15 @@ Resources:
             #!/bin/sh
 
             dd if=/dev/zero bs=300M count=1 | aws s3 cp - s3://${S3ProviderBucket}/300M.dat
+            dd if=/dev/zero bs=512M count=2 > ~/1G.dat
             dd if=/dev/zero bs=512M count=22 | aws s3 cp - s3://${S3ProviderBucket}/11G.dat
+
+            sha256sum ~/1G.dat | awk '{printf "%s",$1}' > ~/1G.dat.sha256
+
+            aws s3 cp ~/1G.dat s3://${S3ProviderBucket}/1G.dat
```

Comment:
Was 1G chosen because the current memory size is 1024?

Reply:
Yeeeeeesss. I thought the memory was 300MB. I did confirm that this test failed before the changes and passed after, so I think it's still valid.